src/main/resources/db/migration/changes/RequestTypes.json

```
@@ -0,0 +1,50 @@
+{
+    "databaseChangeLog": [
+      {
+        "changeSet": {
+          "id": "RequestType-1",
+          "author": "ryanzanone",
+          "preConditions": [
+            {
+              "onFail": "MARK_RAN"
+            },
+            {
+              "not": [
+                {
+                  "tableExists": {
+                    "tableName": "REQUESTTYPES"
+                  }
+                }
+              ]
+            }
+          ],
+          "changes": [
+            {
+              "createTable": {
+                "columns": [
+                  {
+                    "column": {
+                      "autoIncrement": true,
+                      "constraints": {
+                        "primaryKey": true,
+                        "primaryKeyName": "PK_REQUESTTYPES_ID"
+                      },
+                      "name": "ID",
+                      "type": "BIGINT"
+                    }
+                  },
+                  {
+                    "column": {
+                      "name": "REQUESTTYPE",
```

I think this will fail unless you include the snake case version here.

```suggestion
                      "name": "REQUEST_TYPE",
```